tests/functional/d/deprecated/deprecated_method_getmoduleinfo.txt

```diff
@@ -1 +0,0 @@
-deprecated-method:4:::Using deprecated method getmoduleinfo()
```

Comment:
Why not leave this?

Reply:
The maximum test version is 3.6, so when we remove the 3.6 runner this test will never be run again.

Reply:
Sorry, I wasn't clear. I'm questioning why the better solution is not to remove the maximum test version, which looks like it should never have been there in the first place. (pylint warns forever about deprecated stdlib materials, right?)

Reply:
```console
>       self._runTest()
E       AssertionError: Wrong results for file "deprecated_method_getmoduleinfo":
E       
E       Expected in testdata:
E          4: deprecated-method
E       
E       Unexpected in testdata:
E          4: no-member

pylint/testutils/lint_module_test.py:95: AssertionError
```

This is with python 3.10. After `3.6` the method was actually removed so we run into `no-member` messages. I guess that has the same effect as `deprecated-method`?

Reply:
This does sound like we could improve this somewhat indeed!